backend/src/tasks/LookingGlass.ts

```diff
@@ -0,0 +1,259 @@
+import {
+  Organization,
+  Domain,
+  Vulnerability,
+  connectToDatabase
+} from '../models';
+import getIps from './helpers/getIps';
+import { CommandOptions } from './ecs-client';
+import got from 'got';
+import { plainToClass } from 'class-transformer';
+import saveVulnerabilitiesToDb from './helpers/saveVulnerabilitiesToDb';
+import { exit } from 'process';
+import saveDomainsToDb from './helpers/saveDomainsToDb';
+
+async function getAuth() {
+  const results: any[] = await got
+    .post('https://delta.lookingglasscyber.com' + '/auth/login', {
+      json: {
+        params: {
+          username: process.env.LG_USERNAME,
```

Comment:
Just making sure -- do we have to use LG_USERNAME and LG_PASSWORD here? Are we not able to just use the LG_API_KEY to get all necessary data?